tests/gzip/Makefile

```diff
@@ -0,0 +1,41 @@
+# ################################################################
+# Copyright (c) 2017-present, Yann Collet, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+# ################################################################
+
+PRGDIR = ../../programs
+VOID   = /dev/null
+
+
+.PHONY: all
+all: test-gzip-env test-helin-segv test-hufts test-keep test-list test-memcpy-abuse test-mixed test-null-suffix-clobber test-stdin test-trailing-nul test-unpack-invalid test-zdiff test-zgrep-context test-zgrep-f test-zgrep-signal test-znew-k test-z-suffix 
+	@echo Testing completed
+
+.PHONY: zstd
+zstd:
+	$(MAKE) -C $(PRGDIR) zstd
+	#alias gzip='$(PRGDIR)/zstd --format=gzip'
```

Comment:
as far as I understand, this is just for test,
so what about : 
`ln -sf $(PRGDIR)/zstd gzip` ?
It will just create a local link to just built zstd, so nothing disruptive.

Then, I guess it requires a way to force tests to use this local link instead of `/usr/local/bin/gzip`.
Either a variable, or a way to force `./` directory precedence over `/usr/local/bin`